crates/sui-framework-tests/src/unit_tests.rs

```diff
@@ -42,7 +42,7 @@ fn run_deepbook_tests() {
 
 #[test]
 #[cfg_attr(msim, ignore)]
-fn run_examples_move_unit_tests() {
+fn run_sui_framework_examples_move_unit_tests() {
```

Comment:
It's a good idea to rename this, but should this be "programmability" instead of "framework"?

```suggestion
fn run_sui_programmability_examples_move_unit_tests() {
```